Source/WebKit/NetworkProcess/NetworkProcessCreationParameters.h

```diff
@@ -87,6 +87,8 @@ struct NetworkProcessCreationParameters {
     bool ftpEnabled { false };
 
     Vector<WebsiteDataStoreParameters> websiteDataStoreParameters;
+
+    HashSet<String> localhostAliases;
```

Comment:
I wonder whether it would make sense to gate this with `ENABLE(DEVELOPER_MODE)`, because we don't expect this feature to be used at all in production builds, right?